docs/book/how-to/create-and-run-templates/run-a-template.md

```diff
@@ -1,30 +1,51 @@
 ---
-description: Run a template
+description: From a template to a pipeline run
 ---
 
 # Run a template
 
-{% hint style="warning" %}
-This is a [ZenML Pro](https://zenml.io/pro) only feature. Please 
-[sign up here](https://cloud.zenml.io) get access.
+{% hint style="success" %}
+This is a [ZenML Pro](https://zenml.io/pro)-only feature. Please
+[sign up here](https://cloud.zenml.io) to get access.
 {% endhint %}
 
-<!-- ## Run a template from the dashboard -->
+Once your template is set up, you can **run** it and create a new pipeline run 
+that will be conducted on the same stack. Before it starts, you have a window
+to seamlessly readjust the configuration of your pipelines and steps.
 
-## Run a template in code
+## Run a template using the Python SDK
+
+You can use the ZenML client to run a template:
 
 ```python
 from zenml.client import Client
 
-template = Client().get_run_template(<NAME_OR_ID>)
+template = Client().get_run_template("TEMPLATE_NAME")
```

Comment:
I'm not sure why the `<>` was removed, but isn't that at least better to show that it's a placeholder and not just a string?

Reply:
This new format has no Pythonic errors, so I can use my auto-formatting to fix the format of the code even in the docs. Having any `<>`s in the code messes it up completely and it formats in a very weird way. That's why I usually use `""` and capital letters as a convenience. I can change it back though if you want.

Reply:
I mean there is not much to format here 😄 
Or make it "<...>", but all of that is worse IMO, because users will just copy-paste, it will run and then fail with a key error.